client/src/Components/Home/Members/index.jsx

```diff
@@ -1,5 +1,68 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { Card, Row, Col } from 'react-bootstrap';
 
-export default function Members() {
-  return <h1> members cards</h1>;
+import './style.css';
+
+// eslint-disable-next-line react/prop-types
+export default function Members({ filtered, history }) {
+  return (
+    <Row key={Math.random()}>
+      {filtered[0] ? (
+        filtered.map(member => {
+          return (
+            <Col xs={12} md={4} lg={4} key={Math.random()}>
+              <Card
+                className="member-card"
+                key={member.id}
```

Comment:
I don't think we'd need to implicitly set the `key` here since it is wrapped in the `Col` component
The same goes for the rest.